src/main/kotlin/fr/convergence/proddoc/kafka/SurchargeReactive.kt

```diff
@@ -0,0 +1,66 @@
+package fr.convergence.proddoc.kafka
+
+import fr.convergence.proddoc.model.SurchargeDemande
+import fr.convergence.proddoc.model.SurchargeReponse
+import fr.convergence.proddoc.model.lib.obj.MaskMessage
+import fr.convergence.proddoc.service.SurchargeService
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import org.eclipse.microprofile.reactive.messaging.Channel
+import org.eclipse.microprofile.reactive.messaging.Emitter
+import org.eclipse.microprofile.reactive.messaging.Incoming
+import org.slf4j.LoggerFactory
+import java.io.File
+import javax.enterprise.context.ApplicationScoped
+import javax.inject.Inject
+
+@ApplicationScoped
+class SurchargeReactive(
+    @Inject val surchargeService: SurchargeService
+) {
+
+    @Inject
+    @field: Channel("surcharge_fini")
+    val retourEmitter: Emitter<MaskMessage>? = null
+
+    companion object {
+        private val LOG = LoggerFactory.getLogger(SurchargeReactive::class.java)
+    }
+
+    @Incoming("surcharge")
+    fun consume(messageOrigine: MaskMessage) {
+
+        LOG.info("Demande de surcharge")
+
+        var messageRetour: MaskMessage = messageOrigine
+
+        GlobalScope.launch {
+            try {
```

Comment:
tu peux faire 
```suggestion
            retour(try { } catch { })
```
et enlever ce var message = ...

Reply:
ok j'ai changé :)